superset-frontend/cypress-base/cypress/integration/dashboard/drilltodetail.test.ts

```diff
@@ -62,6 +65,41 @@ function setTopLevelTab(tabName: string) {
   cy.get("div#TABS-TOP div[role='tab']").contains(tabName).click();
 }
 
+function testTimeChart(vizType: string) {
+  interceptSamples();
+
+  cy.get(`[data-test-viz-type='${vizType}'] canvas`).then($canvas => {
+    cy.wrap($canvas).scrollIntoView().click(70, 93).rightclick(70, 93);
```

Comment:
I'm assuming the first `click(x, y)` is there because the `rightclick(x, y)` doesn't register without it, as you mentioned was an issue earlier.  To more closely reproduce actual user behavior, how about replacing it with `trigger('mousemove', x, y)`?  I checked locally and it seems to work.

Reply:
It works great. Thanks for the tip!